src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/DocumentTransformer.kt

```diff
@@ -53,9 +43,27 @@ class DocumentTransformer {
       )
     }
 
-    return offenderDocuments + convictionDocuments
+    return offenderDocuments + convertedConvictionDocuments
   }
 
+  fun transformToApi(documents: List<APDeliusDocument>): List<Document> =
+    documents
+      .filter { it.id != null }
+      .map {
+        Document(
+          id = it.id!!,
+          level =  when (it.level) {
+            DocumentLevel.conviction.value -> DocumentLevel.conviction
+            else -> DocumentLevel.offender
+          },
```

Comment:
think this when statement is safe to use `offender` as the fallback as `AP and Delius API` will always give us one or the other. See line 31;
https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/DocumentService.kt